.github/workflows/clp-rust-checks.yaml

```
@@ -2,7 +2,17 @@ name: "clp-rust-checks"
   pull_request:
+    paths: &monitored_paths
+      - ".cargo/**"
+      - "Cargo.lock"
+      - "Cargo.toml"
+      - ".github/workflows/clp-rust-checks.yaml"
+      - "components/clp-rust-utils/**"
+      - "components/log-ingestor/**"
+      - "taskfile.yaml"
+      - "taskfiles/**"
   push:
+    paths: *monitored_paths
```

actionlint (1.7.8)

"paths" section must be sequence node but got alias node with "" tag

(syntax-check)